public_suffix_list.dat

```diff
@@ -14860,11 +14860,39 @@ from.tv
 sakura.tv
 
 // Salesforce.com, Inc. https://salesforce.com/
-// Submitted by Michael Biven <mbiven@salesforce.com> and Aaron Romeo <aaron.romeo@salesforce.com>
+// Submitted by Michael Biven <mbiven@salesforce.com>, Aaron Romeo <aaron.romeo@salesforce.com>, and Steven Lawrance <slawrance@salesforce.com>
 *.builder.code.com
 *.dev-builder.code.com
 *.stg-builder.code.com
 *.001.test.code-builder-stg.platform.salesforce.com
+*.d.crm.dev
+*.w.crm.dev
+*.wa.crm.dev
+*.wb.crm.dev
+*.wc.crm.dev
+*.wd.crm.dev
+*.we.crm.dev
+*.wf.crm.dev
+*.wg.crm.dev
+*.wh.crm.dev
+*.wi.crm.dev
+*.wj.crm.dev
+*.wk.crm.dev
+*.wl.crm.dev
+*.wm.crm.dev
+*.wn.crm.dev
+*.wo.crm.dev
+*.wp.crm.dev
+*.wq.crm.dev
+*.wr.crm.dev
+*.ws.crm.dev
+*.wt.crm.dev
+*.wu.crm.dev
+*.wv.crm.dev
+*.ww.crm.dev
+*.wx.crm.dev
+*.wy.crm.dev
+*.wz.crm.dev
```

Comment:
Is it possible to compress this further? Are all these individual domains necessary?

Reply:
Users-per-line might be too low for acceptance like this.

Reply:
I now have the list cut down to what is presently in operation plus the upcoming ones.

Thanks!